### ISSUE TITLE
Installation issues

### ISSUE THREAD

**Issue by fmarletaz** [ComparativeGenomicsToolkit/cactus#116]
I am trying ton install cactus on a cluster to run locally. I cannot manage to run the example job. The error seems to be with `Received file ID not of type FileID` but I have difficulties interpreting the log further. 
Thanks for your help. 
[cactus.log](https://github.com/ComparativeGenomicsToolkit/cactus/files/3901204/cactus.log)

**Comment by glennhickey** [comment]
Yeah, there's a version of Toil out there (3.21.0 I think) that causes
this.  Reverting back to toil 3.20.0 should fix it.

**Comment by fmarletaz** [comment]
Thanks. That fixed this error. However, now it looks like the the job is stuck. I am trying to run it with singularity instead of docker as docker seems to cause some permission issues. Thanks again. 
[cactus.log](https://github.com/ComparativeGenomicsToolkit/cactus/files/3904392/cactus.log)

**Comment by jasonsydes** [comment]
Your log just shows a single "Reissued any over long jobs" at the end of the log file.  Seeing just one isn't a concern (those will pop up during normal operation).  (For example, it looks like you've seen that message three times in your log file.) 

It's when you see dozens of "Reissued any over long jobs" all next to each other, and nothing else (for example, 24 hours of just those messages) that you should begin to worry.

Did your job move past that point?
